lib/includes/wslay/wslay.h

```diff
@@ -95,6 +95,15 @@ enum wslay_io_flags {
  * sent. If there is an error, return -1. The return value 0 is also
  * treated an error by the library.
  */
+
+#ifdef _MSC_VER
+#ifdef  _WIN64
+typedef int64_t ssize_t;
```

Comment:
It seems you can alternatively use:

```c
#if defined(_MSC_VER)
#include <BaseTsd.h>
typedef SSIZE_T ssize_t;
#endif
```

instead of using `int64_t` or `int32_t` based on the architecture.
That also seem to remove the requirements for the other casts in this PR.